website/docs/reference/project-configs/require-dbt-version.md

```diff
@@ -18,17 +26,19 @@ require-dbt-version: version-range | [version-range]
 
 You can use `require-dbt-version` to restrict your project to only work with a range of dbt versions. 
 
-When you set this configuration, dbt sends a helpful error message for any user who attempts to run the project with an unsupported version of dbt. This can be useful for package maintainers (such as [dbt-utils](https://github.com/dbt-labs/dbt-utils)) to ensure that users' dbt version is compatible with the package. Setting this configuration might also help your whole team remain synchronized on the same version of dbt for local development, to avoid compatibility issues from changed behaviour.
+When you set this configuration, dbt sends a helpful error message for any user who attempts to run the project with an unsupported version of dbt. This can be useful for package maintainers (such as [dbt-utils](https://github.com/dbt-labs/dbt-utils)) to ensure that users' dbt version is compatible with the package. Setting this configuration might also help your whole team remain synchronized on the same version of dbt for local development, to avoid compatibility issues from changed behavior.
```

Comment:
Actually I don't believe that dbt issues an error message for non-matching versions in the "require-dbt-version" at all yet. It issues error messages for packages that have been uploaded to the HUB that have a specified "require_dbt_version" that doesn't match. We probably ought to issue error messages for non-matching dbt_project.yml require-dbt-versions, but it isn't happening yet.